api/src/main/java/org/pmiops/workbench/utils/CohortMapper.java

```diff
@@ -0,0 +1,27 @@
+package org.pmiops.workbench.utils;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.ReportingPolicy;
+import org.pmiops.workbench.db.model.DbCohort;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.Cohort;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {CommonMappers.class},
+    unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface CohortMapper {
+  DbCohort clientToDbModel(Cohort source);
+
+  Cohort dbModelToClient(DbCohort destination);
+
+  default DbUser DbUserToCreatorEmail(String creator) {
+    final DbUser result = new DbUser();
+    result.setEmail(creator);
+    return result;
+  }
+
+  default String DbUserToCreatorEmail(DbUser creator) {
```

Comment:
The inverse is fine since we are going from "lot of data" -> "less data"

Reply:
Are you thinking I should just mark the Creator field as non-mappable? I can do that.

It's still not clear where I would go to find out what fields are required on any of these classes. That's one of the things I'm going to miss about Kotlin if we take it out.